test_conformance/api/test_kernel_arg_changes.cpp

```diff
@@ -15,7 +15,7 @@
 //
 #include "testBase.h"
 
-extern "C" { extern cl_uint gRandomSeed;}
+extern cl_uint gRandomSeed;
```

Comment:
`gRandomSeed` is declared in `testHarness.h`. All other declarations should be removed.

Reply:
Done. I have removed the declarations from other places as well (where `extern "C"` wasn't used). Let me know if this violates some comitting standards.

Reply:
Nope, makes perfect sense. Thanks for doing this!